src/mutex.cpp

```diff
@@ -28,11 +28,6 @@
 
 /* projects.h and windows.h conflict - avoid this! */
```

Comment:
This comment doesn't seem very useful any more. `projects.h` -> `proj_internal.h` or just remove it altogether?